#ifndef IMITATION_NAV__IMITATION_NAV_NODE_HPP_
#define IMITATION_NAV__IMITATION_NAV_NODE_HPP_

#include "imitation_nav/visibility_control.h"

namespace imitation_nav
{

class ImitationNavNode
{
public:
  ImitationNavNode();

  virtual ~ImitationNavNode();
};

}  // namespace imitation_nav

#endif  // IMITATION_NAV__IMITATION_NAV_NODE_HPP_
